store-implementation/parquet-store/src/test/java/uk/gov/gchq/gaffer/parquetstore/operation/handler/spark/AbstractSparkOperationsTest.java

```diff
@@ -137,14 +139,21 @@ public void getElementsWithSeedsRelatedAfterImportElementsFromRDDTestWhenMoreFil
         ElementUtil.assertElementEquals(getResultsForGetElementsWithSeedsRelatedTest(), results);
     }
 
-//    @Test
-//    public void getDataFrameOfElementsTest() throws IOException, OperationException, StoreException {
-//        final Graph graph = genData(false);
-//        final Dataset<Row> data = graph.execute(new GetDataFrameOfElements.Builder()
-//                .build(), user);
-//        checkGetDataFrameOfElements(data, false);
-//    }
-//
+    @Test
+    public void getDataFrameOfElementsTest() throws OperationException {
```

Comment:
Can this be more descriptive eg: `shouldGetDataFrameOfElementsWithNoView()`

Reply:
Done.